package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "osrm",
-  "version": "5.11.0",
+  "version": "5.11.0-latest.1",
   "private": false,
```

Comment:
I will add this to the releasing docs.

What is unclear to me is file compatibility between releases. We bump the package json version but not the CMakeLists version. This results in:
- either we bump to 5.11.0 in CMakeLists immediately after we released 5.10, which would result in the actual 5.11.0 release being compatible with all datasets generated right after we cut 5.10, or
- we only bump to 5.11.0 in CMakeLists right before we publish the proper 5.11.0 release, which would result in all datasets generated before being compatible with 5.10.